### PR TITLE
Fix for subscriptions

### PR DESCRIPTION
**Pull request by PC25**

### CODE REVIEW

**Comment by pradumangoyal** [comment]
I believe this majorly is the replica of https://github.com/IMGIITRoorkee/omniport-service-notifications/pull/11/?

**Comment by PC25** [comment]
> I believe this majorly is the replica of [IMGIITRoorkee/omniport-service-notifications#11](https://github.com/IMGIITRoorkee/omniport-service-notifications/pull/11)?

Should the get_subscription util go into the categories repo?

**Comment by pradumangoyal** [comment]
@PC25 that will be better actually.

File: utils/get_subscription.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+from categories.models import Category, UserSubscription
```
Comment (pradumangoyal) [inline utils/get_subscription.py:1]: We can delete this file, I guess

File: views/subscription.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 from categories.models import UserSubscription, Category
 from categories.serializers import SubscriptionTreeSerializer
+from emails.utils.get_subscription import get_subscription
```
Comment (pradumangoyal) [inline views/subscription.py:10]: Correct the import

**Pull request merged**